test/gui/shared/steps/steps.py

```diff
@@ -1299,11 +1299,10 @@ def step(context):
 @Then('the sync all checkbox should be checked')
 def step(context):
     newAccount = AccountConnectionWizard()
-    test.compare(
-        waitForObjectExists(newAccount.SYNC_DIALOG_ROOT_FOLDER).checkState,
-        "checked",
-        "Assert sync all checkbox is checked",
-    )
+    state = waitForObject(
+        names.deselect_remote_folders_you_do_not_wish_to_synchronize_ownCloud_QModelIndex
```

Comment:
why are we using here the `names` object directly an not the const like `newAccount.SYNC_DIALOG_ROOT_FOLDER` ?